ts/Dashboards/Components/KPIComponent/KPISyncs/KPIExtremesSync.ts

```diff
@@ -0,0 +1,118 @@
+/* *
+ *
+ *  (c) 2009-2024 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Dawid Dragula
+ *
+ * */
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type Sync from '../../Sync/Sync';
+import type KPIComponent from '../KPIComponent.js';
+
+import Component from '../../Component';
+import DataCursor from '../../../../Data/DataCursor';
+import U from '../../../../Core/Utilities.js';
+const { defined } = U;
+
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+const defaultOptions: Sync.OptionsEntry = {};
+
+const syncPair: Sync.SyncPair = {
+    emitter: void 0,
+    handler: function (this: Component): (() => void) | void {
+        if (this.type !== 'KPI') {
+            return;
+        }
+        const component = this as KPIComponent;
+
+        const { board } = this;
+
+        const handleChangeExtremes = (e: DataCursor.Event): void => {
+            const cursor = e.cursor;
+            if (
+                cursor.type === 'position' &&
+                typeof cursor?.row === 'number' &&
+                defined(cursor.column) &&
+                component.connectorHandlers?.[0]?.connector &&
+                !defined(component.options.value)
+            ) {
+                const value = component.connectorHandlers[0].connector
+                    .table.modified.getCellAsString(
+                        cursor.column,
+                        cursor.row
+                    );
+
+                component.setValue(value);
+            }
+
+        };
+
+        const registerCursorListeners = (): void => {
+            const { dataCursor: cursor } = board;
+
+            if (!cursor) {
+                return;
+            }
+            const table = this.connectorHandlers?.[0]?.connector?.table;
```

Comment:
In some places you use `connectorHandlers?.[0]?` in other you use different construction (or use extra condition) like above.

Can we unify it? 

ps. I'm not convinced to the notation: `connectorHandlers?.[0]?` too many `optionals`, so I prefer to add a condition then. Up to you.